test/distributed/fsdp/test_wrap.py

```diff
@@ -479,6 +482,96 @@ def _test_transformer_wrapping(
             else:
                 self.assertFalse(isinstance(module, FSDP))
 
+    @unittest.skipIf(torch.cuda.device_count() < 2, "Requires at least 2 GPUs")
+    def test_lambda_wrap_policy(self):
+        """
+        Tests ``LambdaWrapPolicy`` with both a lambda function that uses
+        uniform kwargs (so only returns ``False`` or ``True``) and a lambda
+        function that uses non-uniform kwargs (so returns a dict to override
+        the root kwargs).
+        """
+        for use_uniform_kwargs in [False, True]:
+            self._test_lambda_wrap_policy(use_uniform_kwargs)
+
+    def _test_lambda_wrap_policy(self, use_uniform_kwargs: bool):
+        model = TransformerWithSharedParams.init(
+            self.process_group,
+            FSDPInitMode.NO_FSDP,
+            CUDAInitMode.CUDA_BEFORE,
+            {},
+        )
+
+        if use_uniform_kwargs:
+
+            def lambda_fn(module: nn.Module):
```

Comment:
does LambdaPolicy still get overriden and individually wrap BatchNorms if mixed precision is configured? 

In general, what happens if my LambdaPolicy tells to wrap a BN, but mixed precision overrides this? What should be the expected behavior?

Reply:
Currently, our mixed precision overrides run last. I am not sure if we should special case the `LambdaWrapPolicy` to be highest priority. What do you think?

I was leaning toward just keeping our overrides as absolute for now.